.travis.yml

```diff
@@ -5,10 +5,13 @@ cache:
   directories:
     - node_modules
 script:
+  - npm run lint
   - npm test
   - npm run build
 deploy:
-  provider: script
-  script: npm run deploy
+  provider: pages
+  skip_cleanup: true
+  github_token: $GITHUB_TOKEN
+  local_dir: dist
   on:
     branch: master
```

Comment:
Great work @fatmali. Just one thing, are we deploying from `master` or the `gh-pages` branch?